tests/test_manage_cyhy_ops.py

```diff
@@ -27,6 +27,18 @@ def test_stdout_version(capsys):
         ), "Standard output by '--version' should agree with module.__version__"
 
 
+def test___main__(capsys):
+    """Verify that the __main__.py file loads correctly."""
+    with pytest.raises(SystemExit):
+        with patch.object(sys, "argv", ["bogus", "--version"]):
+            # cisagov Libraries
+            import manage_cyhy_ops.__main__  # noqa: F401
```

Comment:
There should be a brief comment here explaining why this is `# noqa: F401`.

Reply:
That should probably happen up in `skeleton-python-library`, as that's where the code was added originally.

Reply:
I have added https://github.com/cisagov/skeleton-python-library/issues/54 so that it can get fixed upstream and propagate down.

Reply:
That works- thanks!